Docker/securityserver/files/ss-entrypoint.sh

```diff
@@ -26,4 +26,13 @@ else
     echo "WARN: Installed version ($INSTALLED_VERSION) does not match packaged version ($PACKAGED_VERSION)" >&2
 fi
 
+if [  -z "${XROAD_AUTOLOGIN_PIN}" ]
+then
+    echo "WARN: XROAD_AUTOLOGIN_PIN variable is not set"
```

Comment:
The warning seems a bit noisy since this is an optional method for setting the pin code.

Reply:
Should we change it to INFO: or would you like to remove-it all-together @jhyoty ?

Reply:
I was thinking about removing it altogether, but maybe it could also check if /etc/xroad/autologin is already present and omit the warning/info in that case.